src/components/Plan.js

```diff
@@ -68,20 +83,31 @@ class Plan extends Component {
                         phases.map(phase => {
                             return (
                                 <li className="phase" key={ phase }>{ phase }
-                                    <ul>
-                                        {
-                                            phasedIssues[phase].map(issue =>
-                                                <li className="task" key={ issue.number }>
-                                                    <a href={ issue.url } target="_blank" rel="noopener noreferrer" >{ `${issue.number} ${issue.title}` }</a>
-                                                    <span className={ 'state ' + issue.state }>
-                                                    {
-                                                        issue.state === 'done' ? ' (done)' : issue.state === 'wip' ? ' (in progress)' : ''
-                                                    }
-                                                    </span>
+                                <ul>
+                                    {
+                                        repos.map(repo => {
+                                            if (phasedIssues[phase][repo].length === 0) return null;
+                                            return (
+                                                <li className="repo" key={ repo }>{ repo }
```

Comment:
Hmm, it looks like this will add the extra repo level to the plan even when there's only one repo. Is that desired? (Personally, I think it would be nice to avoid it when there's only one repo.)

Reply:
I changed all the code, so this isn't a thing now